Python/basic GUI calculator

```diff
@@ -0,0 +1,86 @@
+from tkinter import *
+
+def compute_handler():
```

Comment:
It would be better if you can rename the functions according to the operation they perform. i.e - ```def addition()```. As this will help beginners who have no prior  idea of tkinter.

Reply:
sure , thanks for the help